pkg/occlient/templates.go

```diff
@@ -83,15 +83,13 @@ func generateSupervisordDeploymentConfig(commonObjectMeta metav1.ObjectMeta, com
 							Ports: commonImageMeta.Ports,
 							// Run the actual supervisord binary that has been mounted into the container
 							Command: []string{
-								"/opt/odo/bin/dumb-init",
-								"--",
+								"/bin/bash",
```

Comment:
we probably don't need `bash` here. Executing just `/opt/odo/bin/go-init` should be enough.

Reply:
Fixed